### ISSUE TITLE
[webapp] upgrade paramiko

### ISSUE THREAD

**Issue by patricklee2** [Azure/azure-cli#8364]
when paramiko fixes https://github.com/paramiko/paramiko/issues/1369
we want to upgrade cryptography to latest

see https://github.com/Azure/azure-cli/pull/8324

using webapp ssh with ctyptography==2.5.0 will result in CryptographyDeprecationWarning
this is due to paramiko using deprecated  functions

```
/home/patle/azure-cli/env/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:39: CryptographyDeprecationWarning: encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.
  m.add_string(self.Q_C.public_numbers().encode_point())
/home/patle/azure-cli/env/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:96: CryptographyDeprecationWarning: Support for unsafe construction of public numbers from encoded data will be removed in a future version. Please use EllipticCurvePublicKey.from_encoded_point
  self.curve, Q_S_bytes
/home/patle/azure-cli/env/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:111: CryptographyDeprecationWarning: encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.
  hm.add_string(self.Q_C.public_numbers().encode_point())
```

**Comment by patricklee2** [comment]
assign to @patricklee2  and @yugangw-msft

**Comment by patricklee2** [comment]
https://github.com/paramiko/paramiko/issues/1386

**Comment by patricklee2** [comment]
to suppress the warning
```import warnings
warnings.filterwarnings(action='ignore',module='.*paramiko.*')```

**Comment by cowlinator** [comment]
FYI, paramiko has released 2.5.0, which no longer uses deprecated cryptography calls.
https://github.com/paramiko/paramiko/releases/tag/2.5.0
